modules/my_health/spec/requests/my_health/v1/medical_records/allergies_spec.rb

```diff
@@ -15,6 +15,7 @@
   let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
 
   before do
+    Flipper.disable(:mhv_medical_records_migrate_to_api_gateway)
```

Comment:
When dealing with flippers in tests, please use mocking. Otherwise, the enabling/disabling can cause specs to fail when tests are run a parallel (which they are in CI). See [details in this example](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample).

Reply:
Thanks @rmtolmach I will make the fix!

Reply:
@rmtolmach I changed all the Flipper actions over to mocks.